flexget/plugins/sites/freshon.py

```diff
@@ -9,10 +9,21 @@
 from flexget.entry import Entry
 from flexget.event import event
 from flexget.utils.soup import get_soup
+from flexget.utils.tools import parse_filesize
 from flexget.utils.search import normalize_unicode
 
 log = logging.getLogger('search_freshon')
 
+BASE_URL = 'https://freshon.tv'
+SEARCH_FRAGMENT = 'browse.php?search=%s&tab=%s&incldead=%s'
+DL_FRAGMENT = 'download.php?type=rss&id=%s&passkey=%s'
+LOGIN_FRAGMENT = 'login.php?action=makelogin'
```

Comment:
Can't all these "fragments" be replaced with dicts instead? Like you're doing in `login`.